services/mountSecrets.js

```diff
@@ -0,0 +1,37 @@
+const { AzureCliCredential } = require('@azure/identity');
+const { SecretClient } = require('@azure/keyvault-secrets');
+
+async function mount(vaultName, secret) {
+  try {
+    const credential = new AzureCliCredential();
+    const vaultUrl = `https://${vaultName}.vault.azure.net/`;
+    const client = new SecretClient(vaultUrl, credential);
+    const secretValue = await client.getSecret(secret);
+    let secretPrefix = '';
+    let secretSuffix = '';
+
+    if (secret === 'server-key') {
```

Comment:
you can use this regex to avoid the if-else
`const prefix = secretValue.value.match(/-----BEGIN (PRIVATE KEY|CERTIFICATE)-----/gm)[0];
    const sufix = secretValue.value.match(/-----END (PRIVATE KEY|CERTIFICATE)-----/gm)[0];`